src/java_bytecode/java_bytecode_convert_class.h

```diff
@@ -14,6 +17,7 @@ Author: Daniel Kroening, kroening@kroening.com
 
 #include "java_bytecode_parse_tree.h"
 #include "java_bytecode_language.h"
+#include "character_refine_preprocess.h"
```

Comment:
I'm a bit confused by this addition - it appears to stem from this addition: https://github.com/diffblue/cbmc/commit/924e01bd5693f561584ddf67b3ddc77838d698ac but other parts of that commit have not made it across @romainbrenguier could you comment